src/capi/azext_capi/helpers/prompt.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=missing-docstring
```

Comment:
Thanks for the docstrings above! I think it adds a lot for future users of the code, and they can be used to create API documentation.

Could you also do that in this file? 🥺